build.py

```diff
@@ -234,47 +234,43 @@ def clone_or_update_repo(name, branch, dir_name, owner='openwisp'):
         repo_url = f'git@github.com:{repository}.git'
     else:
         repo_url = f'https://github.com/{repository}.git'
-    clone_path = os.path.join('modules', dir_name)
+    clone_path = os.path.abspath(os.path.join('modules', dir_name))
 
     if os.path.exists(clone_path):
         print(f"Repository '{name}' already exists. Updating...")
-        subprocess.run(
-            ['git', 'remote', 'set-branches', 'origin', branch],
-            cwd=clone_path,
-            check=True,
-        )
-        subprocess.run(
-            ['git', 'fetch', '--update-shallow', 'origin', branch],
-            cwd=clone_path,
-            check=True,
-        )
```

Comment:
This was unnecessarily slowing down the CI build. Now, the complete repository is downloaded.